### ISSUE TITLE
Ability to reduce central 'whiteness' in diverging colormaps

### ISSUE THREAD

**Issue by psharda** [1313e/CMasher#49]
I would like to use cmasher's beautiful diverging colormaps for plotting purposes. I need a diverging colormap where the color in the centre is 'close to' white but not completely white. The closest equivalent of what I want is the colormap 'coolwarm' in Matplotlib. Just like cmasher has an ability to cut colors from the edges, is it possible to introduce the ability to cut color from the centre? The available diverging colormaps with a white centre are too white for my purpose. Thanks for considering this idea in advance!

**Comment by 1313e** [comment]
Hmm, that shouldn't be too hard to make. I can have a look at that.

**Comment by psharda** [comment]
Great, thanks! Looking forward to it.

**Comment by DanielYang59** [comment]
This is now possible with the [`combine_cmaps`](https://cmasher.readthedocs.io/user/usage.html#combine-colormaps) utility. It doesn't directly allow chop out a part of a colormap but you could achieve this by taking two sub_cmaps to avoid the central unwanted part and recombine them, like so:
```python
from cmasher.utils import combine_cmaps, get_sub_cmap, view_cmap

cmap_part_0 = get_sub_cmap("cmr.fusion", 0, 0.45)
cmap_part_1 = get_sub_cmap("cmr.fusion", 0.55, 1)

new_cmap = combine_cmaps(cmap_part_0, cmap_part_1)
view_cmap(new_cmap)
```

In this case I removed the central 10% but apparently you would need some further tweaking.

Before:
![before](https://github.com/1313e/CMasher/assets/80093591/0412e5de-298f-4568-ae37-10eb69158c64)

After:
![after](https://github.com/1313e/CMasher/assets/80093591/6a6ee008-a4f3-4c58-b253-d972330267dc)

**Comment by neutrinoceros** [comment]
Thanks @DanielYang59 for your reply ! Indeed it seems that we now have API to support this use case so I believe this can be closed as solved in cmasher 1.8.0. Feel free to ping me if I'm understanding this wrong and the issue should be re-opened !
